release notes/v0.5.0.md

```diff
@@ -0,0 +1,125 @@
+
+  export default function () {
+    const browser = launcher.launch('chromium', {
+      headless: false,
+    });
+    const context = browser.newContext();
+    const page = context.newPage();
+    const res = page.goto('https://test.k6.io/');
+    page.close();
+    browser.close();
+  }
+  ```
+
+- `click` is now an asynchronous method. Note that the `async` keyword is not yet supported. This is a **breaking change** since the behavior now requires the user to take into account concurrent actions. ([#466](https://github.com/grafana/xk6-browser/pull/466))
+
+  Here's an example of how to work with it:
+
+  ```js
+  import { chromium } from 'k6/x/browser';
+
+  export default function() {
+      const browser = chromium.launch();
+      const context = browser.newContext();
+      const page = context.newPage();
+
+      page.goto('https://test.k6.io/', { waitUntil: 'networkidle' });
+      const elem = page.$('a[href="/my_messages.php"]');
+      elem.click().then(() => {
+          page.$('input[name="login"]').type('admin');
+          page.$('input[name="password"]').type('123');
+          return page.$('input[type="submit"]').click();
+      }).then(() => {
+          page.waitForLoadState('networkidle');
+      }).finally(() => {
+          page.close();
+          browser.close();
+      });
+  }
+  ```
+
+  - `waitForNavigation` is now an asynchronous method. Note that the `async` keyword is not yet supported. This is a **breaking change** since the behavior now requires the user to take into account concurrent actions. ([#467](https://github.com/grafana/xk6-browser/pull/467))
+
+  Here's an example of how to work with it:
+
+  ```js
+  import { chromium } from 'k6/x/browser';
+
+  export default function () {
+    const browser = chromium.launch();
+    const context = browser.newContext();
+    const page = context.newPage();
+
+    page.goto("https://test.k6.io/flip_coin.php", {
+      waitUntil: "networkidle",
+    });
+
+    const tails = page.locator("input[value='Bet on tails!']");
+
+    const currentBet = page.locator("//p[starts-with(text(),'Your bet: ')]");
+
+    Promise.all([
+      page.waitForNavigation(),
+      tails.click(),
+    ]).then(() => {
+      console.log(currentBet.innerText());
+    }).finally(() => {
+      page.close();
+      browser.close();
+    })
+  }
+  ```
+
+- Make `fillform.js` example less racy. ([#499](https://github.com/grafana/xk6-browser/pull/499))
+
+## Internals
+
+- Upgraded k6 dependency to v0.40.0. (#?)
+
+- Several refactors to improve maintainability.
+  ([#452](https://github.com/grafana/xk6-browser/pull/452), [#457](https://github.com/grafana/xk6-browser/pull/457), [#464](https://github.com/grafana/xk6-browser/pull/464), [#476](https://github.com/grafana/xk6-browser/pull/476), ([#497](https://github.com/grafana/xk6-browser/pull/497)))
+
+- Add pprof server. ([#503](https://github.com/grafana/xk6-browser/pull/503), [#507](https://github.com/grafana/xk6-browser/pull/507))
```

Comment:
```suggestion
- Add pprof server enabled by an optional environment variable. This will help us troubleshoot issues on a live xk6-browser process. ([#503](https://github.com/grafana/xk6-browser/pull/503), [#507](https://github.com/grafana/xk6-browser/pull/507))
```

Reply:
Done in 74b363a.